code/modules/mob/mob_grab.dm

```diff
@@ -212,9 +212,94 @@
+			var/obj/item/organ/limb/L = victim.get_organ(check_zone(user.zone_sel.selecting))
+			if(victim.health > 0)
+				user.visible_message("<span class='danger'>[user] takes a large bite out of [affecting]'s [L.getDisplayName()]!</span>")
+				victim.apply_damage(15, BRUTE, user.zone_sel.selecting, lingbite_armor_check)
+				attacker.mind.changeling.chem_charges += 0.5
+				playsound(attacker, 'sound/weapons/bite.ogg', 50,)
+			else
+				var/datum/changeling/changeling = attacker.mind.changeling
+				user.visible_message("<span class='danger'>[user] messily rips into [affecting], and begins feeding!</span>")
+				victim.apply_damage(35, BRUTE, user.zone_sel.selecting, lingbite_armor_check)
+				if(victim.disabilities & LING_VICTIM)
+					user <<"<span class='notice'>This form has been ravaged. It is worthless to us.</span>"
+				else
+					if(!do_mob(attacker, victim)||do_after(attacker, 30))
+						if(!changeling.has_dna(victim.dna))
+							changeling.absorb_dna(victim, user)
+							if(attacker.wear_suit)
+								attacker.wear_suit.add_blood(victim)
+								attacker.update_inv_wear_suit(0)
+							else if(attacker.w_uniform)
+								attacker.w_uniform.add_blood(victim)
+								attacker.update_inv_w_uniform(0)
+							if (attacker.gloves)
+								attacker.gloves.add_blood(victim)
+							else
+								attacker.add_blood(victim)
+								attacker.update_inv_gloves()
+							if(attacker.wear_mask)
+								attacker.wear_mask.add_blood(victim)
+								attacker.update_inv_wear_mask(0)
+							if(attacker.head)
+								attacker.head.add_blood(victim)
+								attacker.update_inv_head(0)
+							if(attacker.glasses)
+								attacker.glasses.add_blood(victim)
+								attacker.update_inv_glasses(0)
+							if(attacker.shoes)
+								attacker.shoes.add_blood(victim)
+								attacker.update_inv_shoes(0)
+
+
+							var/obj/item/organ/appendix/A = locate() in victim.internal_organs
+							if(A)
+								qdel(A)
+								user.visible_message("<span class='danger'>[user] tears [affecting]'s appendix out with their teeth and swallows it whole!</span>")
+								changeling.geneticpoints += 2
+								victim.apply_damage(65, BRUTE, "chest", lingbite_armor_check)
+								victim.disabilities |= LING_VICTIM
+								victim.status_flags |= DISFIGURED
+								victim.update_base_icon_state(0)
+							else
+								user.visible_message("<span class='danger'>[user] tears a large chunk from [affecting] and swallows it whole!</span>")
+								changeling.geneticpoints += 1
+								victim.apply_damage(65, BRUTE, "chest", lingbite_armor_check)
+								victim.disabilities |= LING_VICTIM
+								victim.status_flags |= DISFIGURED
+								victim.update_base_icon_state(0)
+
+
+							if(user.nutrition < NUTRITION_LEVEL_WELL_FED)
+								user.nutrition = NUTRITION_LEVEL_WELL_FED
+							if(victim.mind)//if the victim has got a mind
+								victim.mind.show_memory(victim, 0) //I can read your mind, kekeke. Output all their notes.

```

Comment:
pretty sure this is supposed to be:
```victim.mind.show_memory(user, 0)```

At the moment this outputs the victim's notes... to the victim, which is wrong.